.drone.yml

```diff
@@ -156,50 +196,19 @@ steps:
     - refs/tags/*
     event:
     - tag
-
 volumes:
 - name: docker
   host:
     path: /var/run/docker.sock
-
+depends_on:
+- build
 ---
 kind: pipeline
 name: arm
-
 platform:
   os: linux
   arch: arm
-
 steps:
-- name: build
```

Comment:
Any reason arm is removed?

Reply:
I don't think it is.
https://github.com/nickgerace/rancher-fleet/blob/master-windows/.drone.yml#L205-L269